### PR TITLE
Readme: Add 5Ws about faebryk

### PR DESCRIPTION
**Pull request by ruben-iteng**
# Readme: Add 5Ws about faebryk

# Description

Add 5Ws (what, how, who, why, when, where) about faebryk
Fix markdown code blocks

# Checklist

Please read and execute the following:

- [x] My code follows the [coding guidelines](/faebryk/faebryk/blob/main/docs/CODING_GUIDELINES.md) of this project
- [x] My PR title is following the [contribution guidelines](/faebryk/faebryk/blob/main/docs/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I ran [Black](../docs/CONTRIBUTING.md#creating-a-pull-request) to format my code

#### Code of Conduct

By submitting this issue, you agree to follow our [Code of Conduct](/faebryk/faebryk/blob/main/docs/CODE_OF_CONDUCT.md):

- [x] I agree to follow this project's [Code of Conduct](/faebryk/faebryk/blob/main/docs/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
 </div>
 
-## Get running
+---
+
+## About
+
+### What \[is faebryk\]
+
+faebryk is an open-source software-defined EDA (electronic design automation) tool.
```
Comment (NoR8quoh1r) [inline README.md:25]: Normally this would be the other way around: software-defined electronic design automation (EDA) tool.

File: README.md [hunk 2]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
+Think of it like the next step from EDA tools like KiCAD, Altium, Eagle...
+in the way those were the next step from designing electronic circuits on paper.
+While the main focus is on the EDA part currently, faebryk aims to become a wholistic system design tool.
+The main concept of faebryk consists of **describing your system** instead of describing its implementation, and iteratively providing implementation details as needed.
+
+### How \[does faebryk work\]
+
+faebryk itself is just a **python library** that you include in your project. It is providing you with all the tools to describe and design your system and to export that design into something useful like for example a netlist, a devicetree, gerbers etc, which you then can use in the next steps of your project. Key concepts of faebryk are the graph, importers, exporters, the library and the user application.
```
Comment (NoR8quoh1r) [inline README.md:33]: Tools is confusing.
Emphasize instead that the user writes code with faebryk as a library.
Project is overloaded here, the faebryk/software project and the larger product project.
faebryk is just a python library.
You write code describing your (system/product...) using faebryk (primitives/functionality...)
/You write code describing your ... with the help of faebryk.
Once you have created a design, you can export whatever artifacts you need ...

File: README.md [hunk 3]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
+To understand how to use faebryk in your project see #Using faebryk.
+
+### Who \[is faebryk\]
+
+faebryk is a community driven project. That means faebryk does not only consist out of core developers but also users, external contributors, moderators and you! It is founded by a group of embedded, electrical, product design and software engineers with a love for making.
+
+### Why \[do we make faebryk\]
+
+The founders of the project noticed that the innovations of software engineering that make fast, scalable, robust solutions possible have not found their way into hardware design. Furthermore there is a lot of duplicate work done. Think of determining the pinout of a SoC and then having to translate that exact pinout into software again, or having to constantly adapt designs for supply chain issues.
```
Comment (NoR8quoh1r) [inline README.md:42]: We noticed...

File: README.md [hunk 4]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
+Next to that they have assessed that hardware design has quite a big barrier of entry for makers which is not necessary or desirable.
+Currently hardware design is also quite manual labor intensive with very little automation.
+faebryk aims to tackle all those issues and more like proper version management and collaboration.
+
+### When \[is faebryk being developed\]
+
+faebryk is being continuously developed.
+The core team focuses on the core functionality and features of faebryk, and general direction of the project.
+The strength of the community can really shine with the development of importers, exporters, libraries, and projects, but everyone is welcome to help out where they can.
+faebryk uses semantic versioning in the releases.
```
Comment (NoR8quoh1r) [inline README.md:52]: This does not fit in this section. If we add something about stability it would go there.

Comment (IoannisP-ITENG) [inline README.md:52]: add a link to https://semver.org/

File: README.md [hunk 5]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
+faebryk is a community driven project. That means faebryk does not only consist out of core developers but also users, external contributors, moderators and you! It is founded by a group of embedded, electrical, product design and software engineers with a love for making.
+
+### Why \[do we make faebryk\]
+
+The founders of the project noticed that the innovations of software engineering that make fast, scalable, robust solutions possible have not found their way into hardware design. Furthermore there is a lot of duplicate work done. Think of determining the pinout of a SoC and then having to translate that exact pinout into software again, or having to constantly adapt designs for supply chain issues.
+Next to that they have assessed that hardware design has quite a big barrier of entry for makers which is not necessary or desirable.
+Currently hardware design is also quite manual labor intensive with very little automation.
+faebryk aims to tackle all those issues and more like proper version management and collaboration.
+
+### When \[is faebryk being developed\]
```
Comment (NoR8quoh1r) [inline README.md:47]: I don't like this question. Can't think of anything starting with "when", but I don't think we need to be so strict.
How is faebryk developed? seems fine to me.

File: README.md [hunk 6]
```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
...
+
+faebryk is being continuously developed.
+The core team focuses on the core functionality and features of faebryk, and general direction of the project.
```
Comment (NoR8quoh1r) [inline README.md:50]: Suggestions: The core team focuses on core functionality and features of faebryk, as well as the general direction of the project.

**Review by NoR8quoh1r — COMMENTED** [review]
Thanks you for doing this :heart: , expanding the readme in this way adds a lot of value for people checking out faebryk.

I have some comments, consider them suggestions.

**Pull request merged**